ios/keyman/Keyman/SWKeyboard/KeyboardViewController.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  KeyboardViewController.swift
+//  Keyman
+//
+//  Created by Gabriel Wong on 2017-09-04.
+//  Copyright © 2017 SIL International. All rights reserved.
+//
+
+import UIKit
+
+class KeyboardViewController: KMInputViewController {
+  override func updateViewConstraints() {
+    super.updateViewConstraints()
+
+    if view.frame.size.width == 0 || self.view.frame.size.height == 0 {
+      return
+    }
+
+    setupTopBarImage(isPortrait: KMInputViewController.isPortrait())
+  }
+
+  override func viewDidLoad() {
+    KMManager.setApplicationGroupIdentifier("group.KM4I")
+
+    #if DEBUG
+      KMManager.sharedInstance().debugPrintingOn = true
+    #endif
+
+    super.viewDidLoad()
+    setupTopBarImage(isPortrait: KMInputViewController.isPortrait())
+  }
+
```

Comment:
Three or so functions disappeared here, though I noticed that they all directly called their superclass's implementation with no extra details in the Obj-C code.  I figure Swift's automanaging that now, if it's going so far as to automate constructor implementation/use as noted elsewhere.

Reply:
Yep a class inherits all functions from the superclass so I removed those functions. I believe it's the same in Obj-C but the developer copied these functions in from starter code.